src/components/Game/Game.tsx

```diff
@@ -0,0 +1,122 @@
+import React, {useEffect, useState} from 'react';
+import _ from 'lodash'
+
+import './Game.scss';
+
+import FirstGameRow from "../FirstGameRow/FirstGameRow";
+import SecondGameRow from "../SecondGameRow/SecondGameRow";
+import ThirdGameRow from "../ThirdGameRow/ThirdGameRow";
+
+import {randomSum} from "../../services/math-utils";
+import {GameState} from "../../services/CommonTypes";
+
+const Game: React.FC = () => {
+    const MAX_NUMBER_OF_STARS = 9;
```

Comment:
This kind of configuration is best kept in a separate file :) Remember you can import JSONs exactly like you do JS modules.